build/common.ps1

```diff
@@ -492,10 +573,23 @@ Function Invoke-ILMerge {
         [string]$Configuration = $DefaultConfiguration
     )
     $nugetArtifactsFolder = Join-Path $Artifacts "$Configuration\NuGet.Clients\NuGet.CommandLine"
-    pushd $nugetArtifactsFolder
+    $dlls = Get-ChildItem $nugetArtifactsFolder -Filter 'nuget.*.dll' | %{ $_.Name }
```

Comment:
I agree with @emgarten here, we should probably keep the whitelist. The way I have seen done is to have a whitelist+blacklist and fail if a new dll shows up